UT4MasterServer/Controllers/AccountController.cs

```diff
@@ -141,6 +141,29 @@ public async Task<IActionResult> GetAccounts([FromQuery(Name = "accountId")] Lis
 		return Json(arr);
 	}
 
+	[HttpGet("public/accounts")]
+	public async Task<IActionResult> GetAllAccounts()
+	{

```

Comment:
Its fine having this endpoint here for now, but perhaps it would be a better fit in PersonaController